source/stylesheets/header.scss

```diff
@@ -0,0 +1,54 @@
+details > summary {
+	padding: 1.05em;
+	color: white;
+	font-size: 1.8em;
+}
+
+details {
+	background: #ed3e44;
+	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
+}
+
+details[open] {
+	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
+
+}
+
+nav {
+	color: white;
+	position: relative;
+}
+
+details ul {
+	padding: 0;
+	margin: 0;
+}
+
+li {
+ display: block;
```

Comment:
Your indentation is inconsistent in your `scss` files. Choose either 2 or 4 space indentation and reformat these, please!